samples/Server/src/routes/uploadToAzureBlobStorage.ts

```diff
@@ -3,23 +3,28 @@
 
 import * as express from 'express';
 import { getAzureBlobStorageConnectionString } from '../lib/envHelper';
-import { BlobServiceClient, ContainerSASPermissions } from '@azure/storage-blob';
+import { BlobSASPermissions, BlobServiceClient, ContainerSASPermissions, SASProtocol } from '@azure/storage-blob';
+// Alternatively, you can use `multer`, `connect-busboy`, or anything else with `express.JS` to handle file uploads.
```

Comment:
What are the pros and cons of each? Why did we choose to use this one?
Do we need this comment?

Reply:
it wasn't too much difference between them, the one i picked has less code to write, but they are all based on multer and the rest are different extensions based on it.